src/backend/mailbox_manager/management/commands/setup_dimail_db.py

```diff
@@ -64,15 +64,15 @@ def handle(self, *args, **options):
         # this user
         people_base_user = User.objects.filter(name="John Doe")
         if people_base_user.exists():
-            self.create_user(name=people_base_user.sub, password="whatever")  # noqa S106
-            self.create_allows(people_base_user.sub, domain_name)
+            self.create_user(name=people_base_user[0].sub, password="whatever")  # noqa S106
+            self.create_allows(people_base_user[0].sub, domain_name)
 
         self.stdout.write("DONE", ending="\n")
 
     def create_user(
         self,
         name,
-        password,
+        password="no",  # noqa S107
```

Comment:
added dimail's default password when no password are necessary.

Reply:
In dimail `/users/`. I copied it from dimail's tests

https://gitlab.mim-libre.fr/dimail/dimail-api/-/blob/main/src/routes/test_regie.py?ref_type=heads#L74